src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -74,6 +70,13 @@ internal TagHelperExecutionContext(string tagName, TagMode tagMode)
             }
 
             _tagHelpers = new List<ITagHelper>();
+            _allAttributes = new TagHelperAttributeList();
+
+            Context = new TagHelperContext(_allAttributes, items, uniqueId);
+            Output = new TagHelperOutput(tagName, new TagHelperAttributeList(), GetChildContentAsync)
```

Comment:
Have another [issue](https://github.com/aspnet/Razor/issues/720) already logged to fix these things :smile: